### DISCUSSION TITLE
FIXED SPLASH SCREEN

### DISCUSSION THREAD

**Discussion by stanpanchenkosommoio** [buildnatively/community#10] [Ideas]
### Use case

```Text
Hi I really like your platform!
```

### Idea

```Text
The Wrapper is the best by far, but unfortunately there is a problem when you upload an application on top of an existing application, there is a kind of lag in the splash screen that displays the application logo for a second and then displays the splash scene - something that is really disturbing and not fun in the experience of opening the application :(
```

**Comment by galinalf** [comment]
I also don't like the logo that shows before the splash screen. My app was also migrated from another wrapper..

**Comment by romanfurman6** [comment]
Hello @galinalf. To avoid this issue on iOS, consider using the same images on Launch Screen and Loading Screen as shown in the examples above.

![SCR-20240109-ksb](https://github.com/buildnatively/community/assets/22365403/ad7e17df-ecfa-443c-b354-afdaf26bd2ae)
![SCR-20240109-kry](https://github.com/buildnatively/community/assets/22365403/7fd8cce8-6bfc-4ced-b268-782dbbac8423)
